src/pactflow/client/prompts.ts

```diff
@@ -0,0 +1,166 @@
+import { z } from "zod";
+import { PromptParams } from "../../common/types.js";
+
+export const PROMPTS: PromptParams[] = [
+  {
+    name: "OpenAPI Matcher recommendations",
+    params: {
+      description: "Get OpenAPI matcher recommendations using sampling",
+      title: "OpenAPI Matcher recommendations",
+      argsSchema: {
+        openAPI: z.string(),
+      },
+    },
+    callback: function ({ openAPI }: { openAPI: string }): object {
+      return {
+        messages: [
+          {
+            role: "user",
+            content: {
+              type: "text",
+              text: OADMatcherPrompt.replace("{0}", openAPI),
+            },
+          },
+        ],
+      };
+    },
+  },
+];
+
+export const OADMatcherPrompt = `
+
+Generate a list of recommendations(maximum of 5) in JSON to use with an OpenAPI matcher.
+Zod Schema for the matcher to be generated is provided below in the markdown block of javascript use this to generate the recommendations for the matcher. Recommendations should contain all the fields from the schema and only output the JSON in a markdown formatted block.
+
+\`\`\`javascript
+const EndpointMatcherSchema = z
+  .object({
+    path: z
+      .string()
+      .describe(
+        "Path pattern to match specific endpoints (e.g., '/users/{id}', '/users/*', '/users/**'). Supports glob patterns: ? (single char), * (excluding /), ** (including /)"
+      ),
+    methods: z
+      .array(z.enum([
+        "GET",
+        "PUT",
+        "POST",
+        "DELETE",
+        "OPTIONS",
+        "HEAD",
+        "PATCH",
+        "TRACE",
+        ]))
+      .describe(
+        "HTTP methods to include (e.g., ['GET', 'POST']). If not specified, all methods are matched"
+      ),
+    statusCodes: z
+      .array(z.union([z.number(), z.string()]))
+      .describe(
+        "Response status codes to include (e.g., [200, '2XX', 404]). Use 'X' as wildcard (e.g., '2XX' for 200-299). Defaults to successful codes (2XX)"
+      ),
+    operationId: z
+      .string()
+      .describe(
+        "OpenAPI operation ID to match (e.g., 'getUserById', 'get*'). Supports glob patterns"
+      ),
+  })
+  .describe(
+    "REQUIRED: Matcher to specify which endpoints from the OpenAPI document to generate tests for. At least one matcher field must be provided"
+  );
```

Comment:
Wouldn't it be possible to insert the schema here instead of hard-coding code that exists elsewhere in the repository?